tools/distpartitioning/utils.py

```diff
@@ -390,7 +390,7 @@ def write_graph_dgl(graph_file, graph_obj):
     graph_file : string
         File name in which graph object is serialized
     """
-    dgl.save_graphs(graph_file, [graph_obj])
+    dgl.distributed.partition._save_graphs(graph_file, [graph_obj])
```

Comment:
Is this change needed?

Reply:
just to make sure the saved graph has expected dtype.